curvefs/src/mds/fs_manager.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 2021-05-19
+ * Author: chenwei
+ */
+#include <glog/logging.h>
+#include "curvefs/src/mds/fs_manager.h"
+namespace curvefs {
+namespace mds {
+FSStatusCode FsManager::CreateFs(std::string fsName, uint64_t blockSize,
+                curvefs::common::Volume volume, FsInfo* fsInfo) {
+    // 1、查询fs
```

Comment:
should add english comments, change all below.

Reply:
done